servers/xgboostserver/test/test_model_and_predict.py

```diff
@@ -0,0 +1,50 @@
+import os
+import tempfile
+import numpy as np
+from unittest import mock
+import xgboost as xgb
+from XGBoostServer import XGBoostServer
+
+def test_load_json_model():
```

Comment:
could we add a test for the old format as well (if this test doesnt exist elsewhere?)